docs/userGuide/syntax/tags.mbdf

```diff
@@ -1,75 +1,5 @@
-## Tags
-
-You can use tags to selectively filter HTML elements when building a site.
-
-Tags are specified by the `tags` attribute, **and can be attached to any HTML element**. During rendering, only elements that match tags specified in the `site.json` files will be rendered.
-
-<div class="indented">
-
-{{ icon_example }} Attaching tags to elements:
-```html
-# Print 'Hello world'
-
-<p tags="language--java">System.out.println("Hello world");</p>
-<p tags="language--C#">Console.WriteLine("Hello world");</p>
-<p tags="language--python">print("Hello world")</p>
-```
-
-You need to specify the tags to include in `site.json`, under the `tags` option:
-
-```json
-{
-  ...
-  "tags": ["language--java"]
-}
-```
-
-All other tagged elements will be filtered out. In this case, only the element with the `language--java` tag will be rendered. This is helpful when creating multiple versions of a page without having to maintain separate copies.
-
-</div>
-
-If the `tags` option is not specified in `site.json`, all tagged elements will be rendered.
-
-**You can also use multiple tags in a single HTML element. Specify each tag in the `tags` attribute** separated by a space. An element will be rendered if **any of the tags** matches the one in `site.json`.
-
-<div class="indented">
-
-{{ icon_example }} Attaching multiple tags to an element:
-```html
-# For loops
-
-<p tags="language--java language--C#">for (int i = 0; i < 5; i++) { ... }</p>
-```
-
-As long as the `language--java` or `language--C#` tag is specified, the code snippet will be rendered.
-
-</div>
-
-Alternatively, you can specify tags to render for a page in the front matter.
-
-<div class="indented">
-
-{{ icon_example }} Specifying tags in front matter:
-```html
-<frontmatter>
-  title: "Hello World"
-  tags: ["language--java"]
-</frontmatter>
-```
-</div>
-
-Tags in `site.json` will take precedence over the ones in the front matter.
+<include src="../plugins/filterTags.mbdf" />
 
 <span id="short" class="d-none">
-
-```html
-<p tags="language--java advanced">System.out.println("Hello world");</p>
-<p tags="language--C# basic">Console.WriteLine("Hello world");</p>
-```
-```html
-<frontmatter>
-  title: "Hello World"
-  tags: ["language--java"]
-</frontmatter>
-```
-</span>
\ No newline at end of file
+  <include src="../plugins/filterTags.mbdf#short" />
```

Comment:
I did a full-text search and did not find any references using `<include src="syntax/tags.mbdf#short" />`, so this span is not needed?

Also, since `tags.mbdf` is just `<include src="../plugins/filterTags.mbdf" />` , is there a reason to keep this file?

Reply:
A text search will could miss certain relevant nunjucks code e.g., `syntax/{{ file }}.mbdf#short` :-)